nengo_spa/algebras/tvtb_algebra.py

```diff
@@ -1,7 +1,14 @@
+import warnings
```

Comment:
Minor thing but the commit that touches all the `tvtb_algebra.py` stuff is titled "Implement fractional binding powers for VtbAlgebra" when it should be "Implement fractional binding powers for TvtbAlgebra" but that pushes the character limit in the commit name from 50 characters over the limit to 51 characters.